common/scylla_configure.py

```diff
@@ -20,24 +20,22 @@
 import yaml
 import time
 import logging
+import sys
 from lib.log import setup_logging
 from pathlib import Path
-from urllib.request import urlopen
-from urllib.parse import urljoin
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-class ScyllaAmiConfigurator:
+class ScyllaMachineImageConfigurator:
     AMI_CONF_DEFAULTS = {
```

Comment:
If the configurator is now generic, shouldn't `AMI_CONF_DEFAULTS` be renamed to something else?

Reply:
fixed.